src/main/resources/grouper.client.base.properties

```diff
@@ -525,7 +525,7 @@ esb.consumer.encryptionImplementation = edu.internet2.middleware.grouperClient.e
 ## general xmpp configuration
 grouperClient.xmpp.server.host = jabber.school.edu
 grouperClient.xmpp.server.port = 5222
-grouperClient.xmpp.user = username
+grouperClient.xmpp.user = uid
```

Comment:
I don't know if these properties are being used here, but keep it as username because these properties are completely different from the scope of the ticket.